.github/workflows/prerelease-tag.yml

```diff
@@ -0,0 +1,20 @@
+name: "Prerelease"
+
+on:
+  workflow_dispatch:
+    inputs:
+      prerelease_sha:
+        type: string
+        description: "Commit SHA for prerelease tag (default: HEAD main)"
+        required: false
+    
+jobs:
+  prerelease:
+    uses: NVIDIA/NeMo-FW-CI-templates/.github/workflows/_prerelease_tag.yml@ko3n1g/feat/pre-release-tag
```

Comment:
This needs to be changed to a proper version number after all discussions have been resolved